LibertyDSNP/example-client
# Feature/clear data on logout/179661820

Purpose
---------------
[https://www.pivotaltracker.com/story/show/179661820](https://www.pivotaltracker.com/story/show/179661820)
Currently on logout, we clear the session completely and the redux feedSlice but we don't clear profiles or graph. We should clear the entire state.  

Solution
---------------
Should make sure that there are thorough enough tests because this will likely affect that behavior and need fixing.
 
- logging in and back out
- choosing a wallet type and canceling
- logging in with >1 handle
- first time login + registration

We should almost certainly pull the functionality into its own module so it can be tested.

Change summary
---------------
* change login component name to "Connect Wallet" to reflect the words on the button.
* changed class names accordingly
* pulled redux store logout functionality out of the connectWallet component and into its own logout function in a new helpers file in the redux directory.
* added tests for helpers file
* added and refactored tests in connectwallet.test.tsx
* added clearGraph and clearProfiles reducers to the redux store


Steps to Verify
----------------
1. make sure that login / logout / cancel / switch metamask accounts / switch networks all works.
